components/org.wso2.carbon.identity.api.server.claim.management/org.wso2.carbon.identity.rest.api.server.claim.management.v1/src/gen/java/org/wso2/carbon/identity/rest/api/server/claim/management/v1/ClaimManagementApi.java

```diff
@@ -306,18 +335,22 @@ public Response getLocalClaim(@ApiParam(value = "Id of the claim.",required=true
         @io.swagger.annotations.ApiResponse(code = 501, message = "Not Implemented.") })
 
     public Response getLocalClaims(@ApiParam(value = "Define only the required attributes to be sent in the response object.") @QueryParam("attributes")  String attributes,
-    @ApiParam(value = "maximum number of records to return") @QueryParam("limit")  Integer limit,
-    @ApiParam(value = "number of records to skip for pagination") @QueryParam("offset")  Integer offset,
-    @ApiParam(value = "Condition to filter the retrival of records.") @QueryParam("filter")  String filter,
-    @ApiParam(value = "Define the order how the retrieved records should be sorted.") @QueryParam("sort")  String sort)
-    {
-    return delegate.getLocalClaims(attributes,limit,offset,filter,sort);
+    @ApiParam(value = "Maximum number of records to return.") @QueryParam("limit")  Integer limit,
+    @ApiParam(value = "Number of records to skip for pagination.") @QueryParam("offset")  Integer offset,
+    @ApiParam(value = "Condition to filter the retrieval of records.") @QueryParam("filter")  String filter,
+    @ApiParam(value = "Define the order by which the retrieved records should be sorted.") @QueryParam("sort")  String sort,
+    @ApiParam(value = "Exclude identity claims when listing local claims.") @QueryParam("exclude-identity-claims")  Boolean excludeIdentityClaims) {
+
+        return delegate.getLocalClaims(attributes,limit,offset,filter,sort,excludeIdentityClaims);
```

Comment:
There is a parameter addition `excludeIdentityClaims` is this a backward compatible addition?